radio/src/gui/common/stdlcd/radio_sdmanager.cpp

```diff
@@ -277,7 +278,7 @@ void menuRadioSdManager(event_t _event)
             POPUP_MENU_ADD_ITEM(STR_VIEW_TEXT);
           }
 #if defined(LUA)
-          else if (!strcasecmp(ext, SCRIPTS_EXT)) {
+          else if (!strcasecmp(ext, SCRIPT_EXT) || !strcasecmp(ext, SCRIPT_BIN_EXT)) {
```

Comment:
we have the function isImageFileExtension(const char * ext) for images and we could have the same helper for scripts extensions.